tests/cue/src/envars_list.py

```diff
@@ -1,59 +1,56 @@
-py_start = "python3 -c 'import os;"
-
 envars = {
     'VSC_HOME':
     {
-        'name': 'VSC_HOME',
-        'exe': py_start  + """print(os.environ["USER"] in os.environ["{}"])'""",
+        'exe': ['print(os.environ["USER"] in os.environ["VSC_HOME"])'],
     },
     'VSC_DATA':
     {
-        'name': 'VSC_DATA',
-        'exe': py_start  + """print(os.environ["USER"] in os.environ["{}"])'""",
+        'exe': ['print(os.environ["USER"] in os.environ["VSC_DATA"])'],
     },
     'VSC_SCRATCH':
     {
-        'name': 'VSC_SCRATCH',
-        'exe': py_start  + """print(os.environ["USER"] in os.environ["{}"])'""",
+        'exe': ['print(os.environ["USER"] in os.environ["VSC_SCRATCH"])'],
     },
     'VSC_ARCH_LOCAL':
     {
-        'name': 'VSC_ARCH_LOCAL',
-        # needs to have archspec python package installed
-        # 'exe': "py_start  + """import archspec.cpu;print(archspec.cpu.host().name == os.environ["{}"])'""",
+        # archspec is automatically installed as part of ReFrame
+        'exe': [
+            'import archspec.cpu',
+            'aliases = {"skylake": ["skylake_avx512",], "rome": ["zen2"], "milan": ["zen3"]}',  # exceptions to the (full) archspec name
```

Comment:
@Lewih didn't we agree to use zen2/zen3 instead of rome/milan? I think it's better to stick to one name in any case.

Reply:
@backelj ???

Reply:
waiting for @backelj for more info.

Reply:
@smoors update for you: some scripts of ours still require the old naming convention and we are identifying them right now.
We plan to move to the new convention ASAP.
I open an issue to keep track of the problem.